### PR TITLE
hotdoc: Fix has_extensions when several extensions are passed in

### PR DESCRIPTION
**Pull request by thiblahute**
The command we were passing in was wrong.

### CODE REVIEW

File: mesonbuild/modules/hotdoc.py [hunk 1]
```diff
@@ -379,7 +379,7 @@ def __init__(self, interpreter):
 
     @noKwargs
     def has_extensions(self, state, args, kwargs):
-        res = self.hotdoc.run_hotdoc(['--has-extension'] + args) == 0
+        res = self.hotdoc.run_hotdoc(['--has-extension=%s' % extension  for extension in args]) == 0
```
Comment (ignatenkobrain) [inline mesonbuild/modules/hotdoc.py]: __[Flake8]__ <sub>posted by Sider</sub>

[E272] multiple spaces before keyword

File: test cases/frameworks/23 hotdoc/meson.build [hunk 2]
```diff
@@ -7,3 +7,9 @@ endif
 
 subdir('doc')
 
+assert(hotdoc.has_extensions(['gi-extension']) == true,
+  'gi-extension is packaged with core, should always be present')
+
+assert(hotdoc.has_extensions(['gi-extension', 'no-way-you-exist-extension']) == false,
+  'gi-extension is packaged with core, should always be present')
```
Comment (MathieuDuponchelle) [inline test cases/frameworks/23 hotdoc/meson.build]: I don't understand the error message here :)

**Comment by MathieuDuponchelle** [comment]
+1 for me otherwise :)

File: test cases/frameworks/23 hotdoc/meson.build (hunk 2 above)
Comment (thiblahute) [reply test cases/frameworks/23 hotdoc/meson.build]: Better?

Comment (MathieuDuponchelle) [reply test cases/frameworks/23 hotdoc/meson.build]: Yes, brb adding a `no-way-you-exist` extension

**Pull request merged**